app/hooks/use-app-config.ts

```diff
@@ -35,6 +40,7 @@ export const useAppConfig = () => {
           return {
             ...state,
             galoyAuthToken: token,
+            galoySavedAccounts: [...state.galoySavedAccounts, token],
```

Comment:
Should also update the `saveTokenAndInstance` function.

Reply:
Sure